### PR TITLE
Create package

### PR DESCRIPTION
**Pull request by HaLanHuynh**
Create package

LINK: https://edu-redmine.sun-asterisk.vn/issues/10053

### CODE REVIEW

**Comment by HaLanHuynh** [comment]
ready

File: app/src/main/java/com/sun/weather04/screen/history/HistoryFragment.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.sun.weather04.R;
+
+public class HistoryFragment extends Fragment {
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun/weather04/screen/history/HistoryFragment.java]: remove comment

File: app/src/main/java/com/sun/weather04/screen/option/OptionFragment.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.sun.weather04.R;
+
+public class OptionFragment extends Fragment {
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun/weather04/screen/option/OptionFragment.java]: remove comment

File: app/src/main/java/com/sun/weather04/screen/sevendaysnext/SevenDaysNextFragment.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import com.sun.weather04.R;
+
+public class SevenDaysNextFragment extends Fragment {
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun/weather04/screen/sevendaysnext/SevenDaysNextFragment.java]: remove comment

File: app/src/main/res/layout/fragment_history.xml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".screen.history.HistoryFragment"
+    >
+</FrameLayout>
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_history.xml]: enter end line

File: app/src/main/res/layout/fragment_option.xml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".screen.option.OptionFragment"
+    >
+</FrameLayout>
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_option.xml]: enter end line

File: app/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -1,3 +1,6 @@
 <resources>
     <string name="app_name">Weather04</string>
+
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```
Comment (tuanlvt-1117) [inline app/src/main/res/values/strings.xml]: remove string if not use

**Comment by HaLanHuynh** [comment]
ready

**Comment by tuanlvt-1117** [comment]
commented

File: app/src/main/java/com/sun/weather04/screen/history/HistoryFragment.java [hunk 7]
```diff
@@ -0,0 +1,17 @@
+package com.sun.weather04.screen.history;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
```
Comment (thaohq-0814) [inline app/src/main/java/com/sun/weather04/screen/history/HistoryFragment.java]: remove blank

File: app/src/main/java/com/sun/weather04/screen/option/OptionFragment.java [hunk 8]
```diff
@@ -0,0 +1,17 @@
+package com.sun.weather04.screen.option;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
```
Comment (thaohq-0814) [inline app/src/main/java/com/sun/weather04/screen/option/OptionFragment.java]: remove blank

File: app/src/main/res/layout/fragment_daily.xml [hunk 9]
```diff
@@ -0,0 +1,7 @@
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="com.sun.weather04.screen.daily.DailyFragment"
```
Comment (thaohq-0814) [inline app/src/main/res/layout/fragment_daily.xml]: remove context
tiện thể sửa FrameLayout thành ConstraintLayout

File: app/src/main/res/layout/fragment_history.xml [hunk 10]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".screen.history.HistoryFragment"
```
Comment (thaohq-0814) [inline app/src/main/res/layout/fragment_history.xml]: as above

File: app/src/main/res/layout/fragment_home.xml [hunk 11]
```diff
@@ -0,0 +1,7 @@
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="com.sun.weather04.screen.home.HomeFragment"
```
Comment (thaohq-0814) [inline app/src/main/res/layout/fragment_home.xml]: as above

File: app/src/main/res/layout/fragment_hourly.xml [hunk 12]
```diff
@@ -0,0 +1,7 @@
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="com.sun.weather04.screen.hourly.HourlyFragment"
```
Comment (thaohq-0814) [inline app/src/main/res/layout/fragment_hourly.xml]: as above

File: app/src/main/res/layout/fragment_option.xml [hunk 13]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".screen.option.OptionFragment"
```
Comment (thaohq-0814) [inline app/src/main/res/layout/fragment_option.xml]: as above

File: app/src/main/res/layout/fragment_seven_days_next.xml [hunk 14]
```diff
@@ -0,0 +1,7 @@
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="com.sun.weather04.screen.sevendaysnext.SevenDaysNextFragment"
```
Comment (thaohq-0814) [inline app/src/main/res/layout/fragment_seven_days_next.xml]: as above

File: app/src/main/res/layout/fragment_today.xml [hunk 15]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".screen.today.TodayFragment"
```
Comment (thaohq-0814) [inline app/src/main/res/layout/fragment_today.xml]: as above

**Comment by HaLanHuynh** [comment]
ready

**Review by thaohq-0814 — APPROVED** [review]

**Comment by HaLanHuynh** [comment]
ready

**Comment by HaLanHuynh** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**